skeleton-esnext/config.js

```diff
@@ -172,6 +169,10 @@ System.config({
     "npm:bluebird@3.4.1": {
       "process": "github:jspm/nodelibs-process@0.1.2"
     },
+    "npm:bootstrap@4.1.0": {
+      "jquery": "npm:jquery@3.3.1",
+      "tether": "github:HubSpot/tether@1.4.4"
```

Comment:
Bootstrap 4 now use Popper.js instead of Tether. Wondering why it installed that instead?

Reply:
We'll have to get this fixed on JSPM's side.